web/js/map-OpenLayers.js

```diff
@@ -90,7 +90,8 @@ var fixmystreet = fixmystreet || {};
                 size: pin[5] || marker_size,
                 faded: 0,
                 id: pin[3],
-                title: pin[4] || ''
+                title: pin[4] || '',
+                draggable: pin[6] === false ? false : true
```

Comment:
Yep, I was replicating the existing functionality as much as possible, which does allow dragging all markers. Happy to rework it if you want?